### ISSUE TITLE
[FEEDBACK] Windows Setup blocked by Smart Screen

### ISSUE THREAD

**Issue by gittycat123** [lukasbach/yana#19]
Hi,

a very nice app, but the installation is blocked by Windows Defender. 
Maybe upload it to the Windows Store for easier installation for Windows users?
Or disable the false positive report: https://docs.microsoft.com/de-de/windows/security/threat-protection/microsoft-defender-smartscreen/microsoft-defender-smartscreen-overview#%C3%BCbermitteln-von-dateien-an-microsoft-defender-smartscreen-zur-%C3%BCberpr%C3%BCfungsubmit-files-to-microsoft-defender-smartscreen-for-review

Thanks, greetings :)

**Comment by lukasbach** [comment]
Hi, thanks for the feedback! That's interesting, I didn't get anything when I installed it on Windows. Could you maybe provide the error? I'll definitely look into adding the installation for review. Thank you :)

**Comment by lukasbach** [comment]
@gittycat123 Hi, just to get back to the topic, it would definitely be helpful to have the detection name outputted by Windows Defender, because Microsoft wants that name of the detection for uploading the artifact. For some reason, Windows Defender does not trigger a detection on my system.

As for the submission to the Windows Store, I probably will do that at some point in the future, but postpone that for now because Microsoft requires payment for Windows Store submissions.
